### PR TITLE
chore: update gpt2 use case llm multi head attention run with rounding

### PR DESCRIPTION
**Pull request by jfrery**

### CODE REVIEW

File: use_case_examples/llm/QGPT2Evaluate.ipynb [hunk 1]
```diff
@@ -270,24 +271,54 @@
     "output_logits_simulated = proj_single_head_qgpt2(input_ids).logits"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we are ready to execute the single head attention in FHE"
```
Comment (fd0r) [inline use_case_examples/llm/QGPT2Evaluate.ipynb:278]: We shouldn't use "we"

File: use_case_examples/llm/QGPT2Evaluate.ipynb [hunk 2]
```diff
@@ -270,24 +271,54 @@
...
-   "execution_count": 11,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Simulated logits are equal to the quantized clear ones: False\n"
+      "Time taken to execute the single head in FHE: 166.38 seconds\n"
```
Comment (fd0r) [inline use_case_examples/llm/QGPT2Evaluate.ipynb:290]: Yeah with InsertRounding we aren't equal anymore, best to show FHE runtime

**Review by fd0r — CHANGES_REQUESTED** [review]
We should remove the comparison with clear values since they don't apply anymore with rounding.

File: use_case_examples/llm/QGPT2Evaluate.ipynb [hunk 3]
```diff
@@ -270,24 +271,54 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Simulated logits are equal to FHE logits: False\n"
```
[the remark this answers is not in the document]
Comment (jfrery) [reply use_case_examples/llm/QGPT2Evaluate.ipynb:314]: This was correct for exact rounding but now it seems to be wrong with approx :/.

File: use_case_examples/llm/QGPT2Evaluate.ipynb (hunk 1 above)
Comment (jfrery) [reply use_case_examples/llm/QGPT2Evaluate.ipynb]: Removed all the first person pronouns.

File: use_case_examples/llm/QGPT2Evaluate.ipynb (hunk 2 above)
Comment (jfrery) [reply use_case_examples/llm/QGPT2Evaluate.ipynb:290]: We aren't equal to disable (expected) but there seem to be a different between simulate and fhe. Would that be expected?

**Comment by jfrery** [comment]
> We should remove the comparison with clear values since they don't apply anymore with rounding.

I don't compare the prediction with clear values. I compare simulate vs fhe here.

**Comment by fd0r** [comment]
> I compare simulate vs fhe here.

Still doesn't make sense in the approx mode imo

**Review by fd0r — APPROVED** [review]

**Pull request merged**